indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -665,6 +665,7 @@
   {:_id  {:path "concept-id"}}
   {:concept-id (-> m/string-field-mapping m/stored m/doc-values)
    :revision-id (-> m/int-field-mapping m/stored m/doc-values)
+   :concept-seq-id (m/doc-values m/int-field-mapping)
```

Comment:
Can we get rid of the field?

Reply:
Sadly, no -- that's the one I was talking about before: it took me a while to track down a whole slew of errors in the tests, and adding this field was necessary to fix them due to the fact that I was getting errors back from Elastic saying it had no idea how to sort based on a sequence id or something.

I just triple-checked to make sure; without it, tests definitely go crazy.

Reply:
I see all other concepts have this :concept-sea-id for sorting purpose(don't know how it works),  wonder why we didn't have this for the service before...

Reply:
Oh, maybe concept-seq-id means for the same concept-id, different revisions have different concept-seq-id, so it's only needed for the all revision case?